### ISSUE TITLE
Repeatable not working

### ISSUE THREAD

**Issue by biesbjerg** [mattallty/Caporal.js#33]
According to the docs it should possible to specify an option as repeatable to provide more than one value:

Command:
```
./script.js concat -f file1.txt -f file2.txt
```

Code:
```
program
  .version('1.0.0')
  .command('concat') // concat files
  .option('-f <file>', 'File to concat', program.REPEATABLE)
  .action(function(args: any, options: any) {
	console.log(JSON.stringify({args, options}));
  });
```

Expected result
```
{"args":{},"options":{"file": ["file1.txt", "file2.txt"]}}
```

Actual result
```
{"args":{},"options":{}}
```

**Comment by mattallty** [comment]
Thanks for the report @biesbjerg.
Fix on `micromist`dependency should fix this problem, could you please test against `master` ?

**Comment by mattallty** [comment]
Fixed in [v0.5.0](https://github.com/mattallty/Caporal.js/releases/tag/v0.5.0)
